doc/modules/model_evaluation.rst

```diff
@@ -82,15 +82,16 @@ Scoring                      Function                                    Comment
 
 Usage examples:
 
-    >>> from sklearn import svm, cross_validation, datasets
+    >>> from sklearn import svm, datasets
+    >>> from sklearn.model_selection import cross_val_score
     >>> iris = datasets.load_iris()
     >>> X, y = iris.data, iris.target
     >>> model = svm.SVC()
-    >>> cross_validation.cross_val_score(model, X, y, scoring='wrong_choice')
+    >>> cross_val_score(model, X, y, scoring='wrong_choice')
     Traceback (most recent call last):
     ValueError: 'wrong_choice' is not a valid scoring value. Valid options are ['accuracy', 'adjusted_rand_score', 'average_precision', 'f1', 'f1_macro', 'f1_micro', 'f1_samples', 'f1_weighted', 'log_loss', 'mean_absolute_error', 'mean_squared_error', 'median_absolute_error', 'precision', 'precision_macro', 'precision_micro', 'precision_samples', 'precision_weighted', 'r2', 'recall', 'recall_macro', 'recall_micro', 'recall_samples', 'recall_weighted', 'roc_auc']
     >>> clf = svm.SVC(probability=True, random_state=0)
-    >>> cross_validation.cross_val_score(clf, X, y, scoring='log_loss') # doctest: +ELLIPSIS
+    >>> cross_val_score(clf, X, y, scoring='log_loss') # doctest: +ELLIPSIS
     array([-0.07..., -0.16..., -0.06...])
```

Comment:
Please revert the order of the 2 examples: start with the valid use case (`scoring='log_loss'`) first before the failure case (`scoring='wrong_choice'`).